src/Deprecated90/BlockClosure.extension.st

```diff
@@ -0,0 +1,15 @@
+Extension { #name : #BlockClosure }
+
+{ #category : #'*Deprecated90' }
+BlockClosure >> ifError: errorHandlerBlock [
+
+	self deprecated: 'Use #onErrorDo: instead.' transformWith: '`@receiver ifError: `@arg' -> '`@receiver onErrorDo: `@arg'.
```

Comment:
the deprecation should more tricky here which is probably the reason why rewrite was not implemented in first PR.
Originally ifError could accept two arguments (using cull). I do not remember exactly. Check Pharo 8.

I think for simplicity we can do rewrite only for zero argument block. Otherwise use deprecation message.